music21/expressions.py

```diff
@@ -610,18 +798,34 @@ class Mordent(GeneralMordent):
         with the note immediately below it.
 
 
+    A mordent has the size of a second, of some form, depending on the note
+    that will have the mordent, the current key signature in that note's context, as
+    well as any accidental on the mordent itself.
+
     >>> m = expressions.Mordent()
     >>> m.direction
     'down'
-    >>> m.size
-    <music21.interval.GenericInterval 2>
+    >>> m.getSize(pitch.Pitch('C4'))
+    <music21.interval.Interval m-2>
+    >>> m.getSize(pitch.Pitch('B3'))
+    <music21.interval.Interval M-2>
+
+    >>> mFlat = expressions.Mordent(accid='flat')
+    >>> mFlat.direction
+    'down'
+    >>> mFlat.getSize(pitch.Pitch('C4'))
+    <music21.interval.Interval M-2>
+    >>> mFlat.getSize(pitch.Pitch('B3'))
+    <music21.interval.Interval A-2>
 
```

Comment:
can this be demo'd in a key signature context?

Reply:
Done.

Reply:
Oh wait, I did this on a different method.  Not on this one.  Still not done.

Reply:
Done.